apps/subs_web/frontend/webpack.config.js

```diff
@@ -3,6 +3,8 @@ const webpack = require('webpack')
 const ExtractTextPlugin = require('extract-text-webpack-plugin')
 
 module.exports = {
+  mode: 'development',
```

Comment:
Given that we don't have a production config for webpack, is this going to affect the production build?

Reply:
The mode changes the default plugins.

https://webpack.js.org/concepts/mode/

After moving to Webpack 4, a better place to set the mode for dev vs production is in package.json:

"scripts": {
  "dev": "webpack --mode development",
  "build": "webpack --mode production"
}

I removed the changes for Webpack 4.